app/helpers/hosts_helper.rb

```diff
@@ -3,6 +3,7 @@ module HostsHelper
   include HostsAndHostgroupsHelper
   include ComputeResourcesVmsHelper
   include BmcHelper
+  include HostAspectsHelper
 
   def provider_partial_exist?(compute_resource, partial)
```

Comment:
Probably useful to have a test for this method: it implicitly relies on a certain directory structure, and will break should that change. Perhaps pick a couple of most visible compute resources and test those paths?